test/ErrorBoundary.test.js

```diff
@@ -0,0 +1,25 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import ErrorBoundary from '../src/components/ErrorBoundary';
+
+describe('ErrorBoundary Component', () => {
```

Comment:
👍 for grouping (organizing) tests here - good text in `describe` and informative `test` descriptions.

Try doing the same in other tests - to make test report more/easily readable.